packages/approot/console/src/common/utils.ts

```diff
@@ -23,9 +23,9 @@ export function timestampToDate(timestamp: Protobuf.ITimestamp): Date {
 
 /** A sort comparison function for ordering timestamps in ascending progression */
 export function compareTimestampsAscending(a: Protobuf.ITimestamp, b: Protobuf.ITimestamp) {
-  const leftSeconds: Long = a.seconds || Long.fromNumber(0);
+  const leftSeconds = a.seconds! || Long.fromNumber(0);
```

Comment:
not sure if I understand it correctly looking at it at 8pm - as we added bang to `a.seconds!` seems like we expect it always to be present and don't need 'or' part?

Reply:
we get an error for undefined